### PR TITLE
Hotfix/docz downgrade

### PR DESCRIPTION
**Pull request by guilhermedecampo**
Downgrade docz and do some quirks so we can have a working version until the upstream update to a new version. 
The main problem is the typescript docgen parser that's getting all props from styled-components and it's being hard to configure it in a sane manner.

### CODE REVIEW

File: doczrc.js [hunk 1]
```diff
@@ -18,4 +18,6 @@ export default {
   menu: ['Home', 'Basics', 'Components', 'Utilities'],
   files: '**/packages/**/*.mdx',
   dest: '/docs',
+  indexHtml: undefined,
```
Comment (grsabreu) [inline doczrc.js]: out of curiosity, what this does?

File: package.json [hunk 2]
```diff
@@ -4,11 +4,13 @@
   "private": true,
   "homepage": "https://bolt.pismolabs.io/",
   "devDependencies": {
-    "@astrocoders/docz": "0.13.6",
+    "@emotion/cache": "^10.0.0",
```
Comment (grsabreu) [inline package.json]: 🤔

File: packages/timer/Timer.mdx [hunk 3]
```diff
@@ -1,16 +1,12 @@
...
+menu: Components
 ---
 
 import { Playground, PropsTable } from 'docz'
 import { Timer } from './Timer'
 
 # Timer
 
-<PropsTable of={Timer} />
+Uses hooks internally so it won't render nicely here
```
Comment (grsabreu) [inline packages/timer/Timer.mdx]: probably remove then?

File: doczrc.js (hunk 1 above)
Comment (guilhermedecampo) [reply doczrc.js]: it return indeed undefined to indexHtml.. if you don't set nothing there is logic in his code that return `null` in a string. Which the parser tries to parse and break the app. 

I don't know exactly in which version this happened. But I was pretty tired already of having issues and let this there.

File: packages/timer/Timer.mdx (hunk 3 above)
Comment (guilhermedecampo) [reply packages/timer/Timer.mdx]: it's nice to know that this component exists

File: package.json (hunk 2 above)
Comment (guilhermedecampo) [reply package.json]: set of pckages that needs to install to make docz work...

**Pull request merged**